app/views/annotations/update_annotation.js.erb

```diff
@@ -1,7 +1,8 @@
 document.getElementById('annotation_summary_list').innerHTML =
-  '<%= escape_javascript(render partial: "results/marker/annotation_summary",
+  '<%= escape_javascript(render partial: "results/common/annotation_summary",
                                 locals: { annots: @annotations,
-                                          submission_file_id: @submission_file_id }) %>';
+                                          submission_file_id: @submission_file_id,
+                                          marker: true}) %>';
```

Comment:
I only wanted you to combine the views related to the criteria types. Please revert all other changes you made.